Sign_Up_Demo/View/UserView.swift

```diff
@@ -26,43 +26,59 @@ class UserView: UIView {
     @IBOutlet weak var btnSignUp: UIButton!
     @IBOutlet weak var lblInstructions: UILabel!
     
-    var myMutableString = NSMutableAttributedString()
-    var style = NSMutableParagraphStyle()
+    
     
     override func awakeFromNib() {
         super.awakeFromNib()
         createSubviews()
     }
-    
+}
+
+extension UserView {
     func createSubviews() {
```

Comment:
Always try to restrict the method, properties etc with maximum restricted access modifier (public, private etc)